tutorials/mlx-array-framework-apple-silicon/index.mdx

```diff
@@ -0,0 +1,103 @@
+  paragraph: This page provides information how to install and use the MLX array framework on Scaleway's Apple silicon as a Service.
+categories:
+  - compute
+tags: apple-silicon mlx framework apple mac-mini
+dates:
+  validation: 2023-12-15
+  posted: 2023-12-15
+---
+
+
+MLX, an array framework designed for effective and versatile machine learning on Apple silicon, was developed by Apple’s machine learning research team.
+MLX is developed by machine learning researchers focusing on catering to their peers, emphasizing a balance between user-friendliness and efficiency in model training and deployment. The framework boasts a deliberately straightforward design that facilitates seamless extension and enhancement by researchers, fostering the swift exploration of innovative ideas.
+Regarding APIs, the Python interface of MLX closely emulates NumPy, incorporating a few distinctive features. MLX also offers a robust C++ API designed closely with its Python counterpart.
+Taking inspiration from frameworks such as PyTorch, Jax, and ArrayFire, MLX takes a unique path with its unified memory model. Within MLX, arrays reside in shared memory, allowing operations on MLX arrays across diverse supported devices without data copies. This approach marks a notable distinction from other frameworks.
+
+Key features of MLX include:
+
+* **Familiar APIs:** MLX provides a Python API closely aligned with NumPy, complemented by a comprehensive C++ API closely mirroring its Python counterpart. Higher-level packages like `mlx.nn` and `mlx.optimizers` closely follow PyTorch, simplifying the construction of intricate models.
+* **Composable Function Transformations:** MLX facilitates composable function transformations, supporting automatic differentiation, vectorization, and optimization of computation graphs.
+* **Lazy Computation:** Within MLX, computations adopt a lazy approach, only materializing arrays when needed.
+* **Dynamic Graph Construction:** MLX dynamically constructs computation graphs, allowing for shape changes in function arguments without triggering sluggish compilations. Debugging remains straightforward and intuitive.
+* **Multi-device Support:** MLX operations are versatile and capable of running on supported devices, currently spanning the CPU and GPU.
+* **Unified Memory:** A standout distinction lies in MLX's unified memory model. Arrays within MLX reside in shared memory, permitting operations on arrays across different device types without necessitating data transfers.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created an Mac mini](/bare-metal/apple-silicon/how-to/create-mac-mini/)
+  - You have native Python >= 3.8 installed on the Mac (preinstalled by default)
+</Message> 
+
+<Message type="note">
+    MLX is only available on devices running MacOS >= 13.3 It is highly recommended to use MacOS 14 (Sonoma) or later.
```

Comment:
We don't yet preinstall Sonoma by default, this may make the user experience a little complicated, what do you think?

Reply:
I think we can remove this sentence then

Reply:
```suggestion
    MLX is only available on devices running MacOS >= 13.3.
```